examples/two-columns-layout/src/searchClient.ts

```diff
@@ -0,0 +1,8 @@
+import algoliasearch from 'algoliasearch';
```

Comment:
This is a lighter client with only the search-related methods.

```suggestion
import algoliasearch from 'algoliasearch/lite';
```